app/src/main/java/org/kiwix/kiwixmobile/KiwixMobileActivity.java

```diff
@@ -425,6 +426,15 @@ public void onCreate(Bundle savedInstanceState) {
 
     ActionBarDrawerToggle drawerToggle = new KiwixActionBarDrawerToggle(this, drawerLayout, toolbar);
 
+    SharedPreferences firstRunPref = PreferenceManager.getDefaultSharedPreferences(this);
```

Comment:
Please check the newly created ```SharedPreferenceManager``` and use it.

Reply:
SharedPreferenceUtil?

Reply:
Yes. Sorry for late reply